av1-rtp-spec.md

```diff
@@ -373,6 +373,19 @@ described in [I-D.ietf-mmusic-sdp-simulcast][I-D.ietf-mmusic-rid]. Alternatively
 be transported on a single RTP stream, in which case Restriction Identifiers (RIDs) are not used. In either
 case, simulcast transport MUST only be used to convey multiple encodings from the same source.
 
+### 6.1.1 Example
+
+In this example, it is desired to send three simulcast encodings, each containing three temporal
+layers. When sending all encodings on a single SSRC, scalability mode 'S3T3' would be indicated
+within the scalability metadata OBU, and the dependency descriptor describes the dependency structure

```

Comment:
I doubled checked my concerns. Current text looks good to me.

Section 5 of the rtp spec limits operating points described in sequence header OBU (section 5.5 in main av1 spec)
This text only mentions scalability metadata OBU (section 5.8.5 of the main av1 spec).
I didn't remember that obu exists when I raised my concerns.
The metadata OBU is unrestricted and doesn't conflict with limits on the operating points: it is ok to have S3T3 scalability mode and single operating point 0xfff (and do filtering using the dependency descriptor)

Reply:
Yes. WebRTC implementations do not support reception of simulcast with any transport, so the SFU would need to only forward a single encoding on to the receiver.  However, if the RTP payload was encrypted the SFU could not modify the scalability metadata OBU, so it would still be S3T3 even though the forwarded stream would only correspond to L1T3. This seems potentially problematic because there would be holes in the sequence space.  This might cause the receiver to send NACKs for the missing RTP packets, even though the SFU chose not to forward them (e.g. holes not caused by loss).